core/cli_triage_apply.py

```
@@ -0,0 +1,152 @@
+
+def _project_exists(payload: dict, project_name: str) -> bool:
+    clean = project_name.strip().lower()
+    for p in payload.get("projects", []):
+        if str(p.get("name", "")).strip().lower() == clean:
+            return True
+    return False
+
+
+@app.command("triage-apply")
+def triage_apply(
+    input: Annotated[
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)